Source/DataSources/KmlDataSource.js

```diff
@@ -1474,27 +1635,298 @@ define([
         console.log('KML - Unsupported feature: ' + node.localName);
     }
 
+    var RefreshMode = {
+        INTERVAL : 0,
+        EXPIRE : 1,
+        STOP : 2
+    };
+
+    var scratchCartographic = new Cartographic();
+    function computeViewRectangle(camera) {
+        if (!defined(camera)) {
+            return Rectangle.MAX_VALUE;
+        }
+
+        var region = camera.computeViewRegion();
+        if (!defined(region)) {
+            return Rectangle.MAX_VALUE;
+        }
+
+        var result;
+        var ellipsoid = Ellipsoid.WGS84;
+        var count = region.length;
+        for(var i=0;i<count;++i) {
```

Comment:
Can you run the auto-formatter on this file.  You can just do the whole file and then run `npm run sortRequires` to fix the heading. (also submit any other files that `sortRequires` updates.)